src/app/learn/reference/component-libraries/index.html

```diff
@@ -177,4 +177,24 @@
     }
   </sky-code-block>
 
+  <stache-page-anchor>
+    Test modules
+  </stache-page-anchor>
+
+  <p>
+    You can generate a separate module for testing by adding a test directory. The test module uses a separate test file to prevent cluttering the production module during webpack builds.
```

Comment:
In the first sentence, should the subject be "Libraries" instead of "You"? I _think_ so in the sense that "you" set up directory but that the library itself seems to generate the module. Also, it seems like "an isolated" might be more precise and meaningful than "a separate." Also, Steve noted in the PR for this feature that consuming SPAs use the testing module for unit tests. It seems like maybe we should include this point. (I'd also capitalize "Webpack.") What do you think of rearranging like this:

"Libraries can generate isolated test modules for consuming SPAs to use in unit tests. The testing module avoids cluttering the production module during Webpack builds."